app/services/audit_log_service.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AuditLogService
+
+  def log(message)
+    Rails.logger.info "[AUDIT]: #{message}"
+  end
+end
```

Comment:
This looks good. Please follow a service class pattern as described here:
https://github.com/selleo/pattern#service

You basically need to `include Callable` and then log like `AuditLogService.call(my_message)`.